src/core/agent/services/multiSigService.ts

```diff
@@ -555,7 +555,7 @@ class MultiSigService extends AgentService {
       rot: [serder, atc],
     };
 
-    const smids = states.map((state) => state["i"]);
+    const rmids = states.map((state) => state["i"]);
```

Comment:
Actually, `rotateMultisig` calls `const multisigMembers = members?.signing;` to get these `multisigAidMembers`. We need to check `members?.rotation` and pass that here for `rmids`.

From what I can tell, we convert `members?.signing` (`smids`) to an `Aid` type, and then convert it back after...? Can you check if we can change `rotateMultisigAid` to accept `smids` and `rmids` (`string[]`) instead of `multisigAidMembers`.

And in `rotateMultisig` just pass: `smids: members.signing, rmids: members.rotation`